src/app/pages/send/ordinal-inscription/hooks/use-send-inscription-fees-list.ts

```diff
@@ -36,63 +62,47 @@ export function useSendInscriptionFeesList({ recipient, utxo }: UseSendInscripti
 
     if (!feeRates || !nativeSegwitUtxos || !nativeSegwitSigner) return [];
 
-    const highFeeResult = selectInscriptionTransferCoins({
-      recipient,
-      inscriptionInput: utxo,
-      nativeSegwitUtxos,
-      changeAddress: nativeSegwitSigner.payment.address!,
-      feeRate: feeRates.fastestFee.toNumber(),
-    });
-
-    const standardFeeResult = selectInscriptionTransferCoins({
-      recipient,
-      inscriptionInput: utxo,
-      nativeSegwitUtxos,
-      changeAddress: nativeSegwitSigner.payment.address!,
-      feeRate: feeRates.halfHourFee.toNumber(),
-    });
-
-    const lowFeeResult = selectInscriptionTransferCoins({
-      recipient,
-      inscriptionInput: utxo,
-      nativeSegwitUtxos,
-      changeAddress: nativeSegwitSigner.payment.address!,
-      feeRate: feeRates.hourFee.toNumber(),
-    });
-
-    if (!highFeeResult.success || !standardFeeResult.success || !lowFeeResult.success) return [];
-
-    const { txFee: highFeeValue } = highFeeResult;
-    const { txFee: standardFeeValue } = standardFeeResult;
-    const { txFee: lowFeeValue } = lowFeeResult;
-
-    return [
-      {
+    const highFeeValue = getTransactionFee(feeRates.fastestFee.toNumber());
+    const standardFeeValue = getTransactionFee(feeRates.halfHourFee.toNumber());
+    const lowFeeValue = getTransactionFee(feeRates.hourFee.toNumber());
+
+    const feesArr = [];
+
+    if (highFeeValue) {
```

Comment:
This seems almost the same as - https://github.com/leather-wallet/extension/pull/5011/files#r1514527890 

Maybe we can extract and share logic?

Reply:
I will change the fees gen logic in future prs, no much sense in refactoring this now